pkg/validate/validate.go

```diff
@@ -72,7 +72,7 @@ func actionSpec(s crv1alpha1.ActionSpec) error {
 		// Not a known type. ActionSet must specify API group and resource
 		// name in order to populate `Object` TemplateParam
 		if s.Object.APIVersion == "" || s.Object.Resource == "" {
-			return errorf(validateErr, "Not a known object Kind %s. Action %s must specify Resource name and API version", s.Object.Kind, s.Name)
+			return errorf(errValidate, "Not a known object Kind %s. Action %s must specify Resource name and API version", s.Object.Kind, s.Name)
```

Comment:
does `validateErr` not pass but `errValidate` pass?

Reply:
yes, aacording to stylecheck lint(ST1012), it says `error var validateErr should have name of the form errFoo (stylecheck)`